packages/web-app-files/src/components/Sidebar.vue

```diff
@@ -217,4 +219,7 @@ export default {
     fill: #ffba0a !important;
   }
 }
+#files-sidebar-item-name {
+  font-size: medium;
```

Comment:
While I like the readability of this I'd suggest we stick with either `rem` or our custom css properties (`--oc-font-size-default` in this case I guess) for consistancy?

Reply:
done 🥳